packages/perseus-editor/src/item-extras-editor.tsx

```diff
@@ -26,18 +26,24 @@ class ItemExtrasEditor extends React.Component<Props, State> {
         zTable: false,
     };
 
+    finCalcOptions = [
+        this.props.financialCalculatorMonthlyPayment,
+        this.props.financialCalculatorTotalAmount,
+        this.props.financialCalculatorTimeToPayOff,
+    ];
```

Comment:
This makes me a bit nervous. We're accessing props when instantiating a class field.  I know in the past we've had "race"-type errors where `this` is not properly set up during field initialization.  See: https://github.com/microsoft/TypeScript/issues/52331 (I'm not sure it's still an issue in our version of Typescript, but I think it's worth avoiding this pattern). 

Could we move this, and the `state` initialization into a constructor?